tensorflow-examples/src/main/java/org/tensorflow/model/examples/cnn/fastrcnn/FasterRcnnInception.java

```diff
@@ -269,7 +270,6 @@ public static void main(String[] params) {
                 );
                 //fresh runner for reshape
                 runner = s.runner();
-                s.run(tf.init());
```

Comment:
I think you should fold the runner into the try catch below so it would be `try (TUint8 reshapeTensor = (TUint8) s.runner().fetch(reshape).run().get(0)) {`

Reply:
This was also done and committed

Reply:
I think I missed moving the s.runner() into try catch - I've moved that now.